bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsVoTableCatalog.scala

```diff
@@ -26,7 +26,7 @@ import jsky.util.gui.StatusLogger
  * The catalog search will provide the inputs to the analysis phase, which actually assigns guide stars to guiders.
  * See OT-26
  */
-case class GemsVoTableCatalog(backend: VoTableBackend = RemoteBackend) {
+case class GemsVoTableCatalog(backend: VoTableBackend = RemoteBackend, catalog: CatalogName = ucac4) {
```

Comment:
What is the reasoning behind selecting a default catalog (and back end) in the constructor? I generally prefer a companion method with an irritating name like `.forRemoteUcac4` for this kind of thing.